app/src/scenes/Items/index.tsx

```diff
@@ -0,0 +1,121 @@
+} from './interfaces';
+import { List } from 'antd';
+import { RouteComponentProps } from 'react-router';
+import styles from './styles.css';
+
+class Items extends React.Component<
+  RouteComponentProps<CategoryUrlParams>,
+  ItemStateProps
+> {
+  pageSize: number;
+  SubCategoryName: string;
+  SubCategoryId: string;
+  constructor(props: RouteComponentProps<any>) {
+    super(props);
+    this.pageSize = 10;
+    this.SubCategoryName = '';
+    this.SubCategoryId = this.props.match.params.subCategoryId;
+    console.log(this.SubCategoryId);
+    this.state = {
+      isLoading: false,
+      items: [],
+      pagination: {
+        current: 1,
+        total: 0,
+        pageSize: this.pageSize,
+      },
+    };
+  }
+
+  componentDidMount() {
+    this.fetchItems(1);
+    this.fetchCategoryDetails();
+  }
+
+  fetchCategoryDetails = () => {
+    axios
+      .get(
+        window.location.origin +
+          `/core/academix/sub-categories/${this.SubCategoryId}`
+      )
+      .then((result: AxiosResponse<SubCategory>) => {
+        if (result.status == 200) {
+          this.SubCategoryName = result.data.translations[0].name;
+        }
+      })
+      .catch((error) => {
+        handleApiError(
+          error,
+          'Something went wrong when trying to load category details'
+        );
+      });
+  };
+
+  fetchItems = (pageNo: number) => {
+    const pageNumber = pageNo - 1;
+    this.setState({ isLoading: true });
+    axios
+      .get(
+        window.location.origin +
+          `/core/academix/sub-categories/1/items?pageNumber=${pageNumber}&pageSize=${this.pageSize}`
+      )
+      .then((result: AxiosResponse<ItemPayload>) => {
+        if (result.status == 200) {
+          const pagination = { ...this.state.pagination };
+          pagination.current = pageNo;
+          pagination.total = result.data.totalElements;
+          this.setState({
+            isLoading: false,
+            items: result.data.content,
+            pagination: pagination,
+          });
+        }
+      })
+      .catch((error) => {
+        handleApiError(error, 'Something went wrong when trying to load items');
+        this.setState({ isLoading: false });
+      });
+  };
+
+  handleListChange = (currentPage: number) => {
+    if (currentPage != undefined) {
+      this.fetchItems(currentPage);
+    }
+  };
+
+  render() {
+    return (
+      <List
+        className={styles.mainContent}
+        pagination={{
+          onChange: (currentPage: number) => {
+            this.handleListChange(currentPage);
+          },
+          pageSize: this.pageSize,
+          total: this.state.pagination.total,
+        }}
+        loading={this.state.isLoading}
+      >
+        <h1>{this.SubCategoryName}</h1>
+        {this.state.items.map((item) => (
+          <List.Item.Meta
+            key={item.id}
+            title={<a href={item.link}>{item.translations[0].name}</a>}
+            description={item.translations[0].description}
```

Comment:
Better if we could remove the description because it can be a lengthy one. We can display the description on the modal.